packages/h/honymce.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "honymce",
+  "filename": "tinymce.min.js",
+  "description": "HonyMce rich text editor, Personalized By Milad",
+  "homepage": "https://github.com/miladshakerdn/honymce",
+  "keywords": [
+    "tinymce",
+    "editor",
+    "wysiwyg"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/miladshakerdn/honymce.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/miladshakerdn/honymce.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.min.+(css|js|map)",
+          "skins/lightgray/img/*",
+          "skins/lightgray/fonts/*",
+          "plugins/**/plugin.js",
+          "plugins/**/i18n/**/*.js",
+          "plugins/**/*.!(js)",
+          "Langs/**/*.!(js)"
```

Comment:
I'm a bit confused by the files you're proposing here -- you have a pretty open glob-star on the first line, and then a load of more specific stuff that is already going to be covered by the glob-star? You also seem to be referencing directories that don't exist, such as `skins/lightgray` and `Langs`?